RunningOrder/Sprint/Managers/SprintManager.swift

```diff
@@ -15,13 +15,15 @@ final class SprintManager: ObservableObject {
     @Published var sprints: [Sprint] = []
 
     @Stored(fileName: "sprints.json", directory: .applicationSupportDirectory) private var storedSprints: [Sprint]?
+    @Published var isLoading = false
```

Comment:
please remove this. as it was your previous implementation, dead code :)

Reply:
you dead

Reply:
☠️